tools/jetpack-live-branches/jetpack-live-branches.user.js

```diff
@@ -18,6 +18,25 @@
 	const markdownBodySelector = '.pull-discussion-timeline .markdown-body';
 	let pluginsList = null;
 
+	const style = document.createElement( 'style' );
+	style.innerHTML = `
+		#jetpack-live-branches .optionslist {
+			list-style: none;
+			padding-left: 0;
+			margin-top: 24px;
+			display: flex;
+			flex-wrap: wrap;
+		}
+
+		#jetpack-live-branches label {
+			font-weight: inherit;
+		}
+
+		#jetpack-live-branches label.disabled {
+			color: rgba(0,0,0,0.5);
```

Comment:
```suggestion
			color:#7d8590;
```

I would suggest a different color choice to work better with dark mode because it looks strange this way:

<img width="816" alt="Screenshot 2023-07-13 at 14 48 01" src="https://github.com/Automattic/jetpack/assets/1242807/08f62d5d-877e-4a78-ab72-7443b9551e64">

Reply:
Good idea, but it looks like we can go one step further and try to follow GitHub's own theming with something like
```suggestion
			color: var( --color-fg-muted, #7d8590 );
```
Not sure whether we should fall back to the current light theme value (#656d76) or dark theme value (#7d8590) should GH ever rename their variable. Or maybe halfway in between (#717983)?

Reply:
I tried the #7d8590 with a light and dark theme, and it works well either way. It's a small thing to change if we find things don't work.